main.go

```diff
@@ -221,58 +286,37 @@ func remove(c *gin.Context) {
 		c.JSON(http.StatusBadRequest, gin.H{"result": "invalid request JSON"})
 	}
 
-	// Override the name due to our validation mechanism
-	// TODO: Remove this override in favor of a different validation mechanism
-	serverData.Name = "not important"
-
-	fmt.Println("A server is registering.")
-
-	if validateEntry(c, serverData.Name, serverData.IP.String(), strconv.Itoa(serverData.Port)) {
-		// key is your incoming IP:advertised port (string)
-		key := createKey(serverData.IP.String(), strconv.Itoa(serverData.Port))
-
-		mux.RLock()
-		if _, ok := servers[key]; ok {
-			// It was found, remove it.
-			fmt.Println("This server is being removed.")
-			c.JSON(200, gin.H{"result": "success"})
-		} else {
-			// Report back to user: it wasn't removed because it wasn't found.
-			fmt.Println("The server was not found.")
-			c.JSON(404, gin.H{"result": "failure"})
-		}
-		mux.RUnlock()
+	fmt.Println("A server is being removed.")
 
-		mux.Lock()
-		delete(servers, key)
-		mux.Unlock()
-	} else {
+	// Hard-code the name due to our validation mechanism
+	// TODO: Remove this override in favor of a different validation mechanism
+	serverName := "not important"
```

Comment:
That was just a hack for the moment, but I'd be interested to figure out how to get rid of this little placeholder. The validateEntry function probably doesn't always need to validate the name, but I'm not sure of the cleanest way to do this. It would change the function signature.

Reply:
Lets get this resolved in this PR, then we can merge the whole shagang.

Reply:
Ah, yea, if you noticed the TODOs it's because validation is something I wanted to work on next. I just didn't want to bloat the PR so that we can iterate a bit faster.

But yea, we can change the validation interface so that it doesn't need this workaround. 😃